state/mocks/block_store.go

```diff
@@ -1,4 +1,4 @@
-// Code generated by mockery 2.7.5. DO NOT EDIT.
+// Code generated by mockery v0.0.0-dev. DO NOT EDIT.
```

Comment:
What version of mockery are you running lol?

Reply:
Probably whatever they have in master. Can fix.

Reply:
So, it turns out this is an issue in mockery. See: https://github.com/vektra/mockery/issues/397

 When running in modules mode, mockery does not set the string correctly. I'm going to open a separate PR to add and update the mocks so this will be a little smoother.

Reply:
Updated the mocks in #6787 . `make mockery` will now use the same version of mockery across all developer machines.